www/class/CentreonLDAPAdmin.class.php

```diff
@@ -578,14 +564,14 @@ public function getLdapConfigurationList($search = "", $offset = null, $limit =
     public function deleteConfiguration($configList = [])
     {
         if (count($configList)) {
-            $statement = $this->db->prepare(
-                "DELETE FROM auth_ressource
-                WHERE ar_id
-                IN (:configList)"
-            );
 
-            $statement->bindValue(':configList', implode(',', $configList), \PDO::PARAM_STR);
-            $statement->execute();
+            foreach ($configList as $val) {
+                $statement = $this->db->prepare(
+                    "DELETE FROM auth_ressource WHERE ar_id IN (:configList)"
+                );
+                $statement->bindValue(':configList', $val, \PDO::PARAM_INT);
+                $statement->execute();
+            }
```

Comment:
are you sure that it is not the key which is used ?

Reply:
reverted